pkgs/applications/networking/mailreaders/mailnag/default.nix

```diff
@@ -31,6 +33,10 @@ pythonPackages.buildPythonApplication rec {
     pygobject3 dbus-python pyxdg
   ];
 
+  buildPhase = "";
+
+  installPhase = "python2 setup.py install --prefix=$out";
```

Comment:
Is this really needed? Our default builder builds wheels and installs those.

Reply:
@FRidh Without these it doesn't work correctly. cc @bkchr @dezgeg https://github.com/gnidorah/nixpkgs/commit/5b91f125aa58fb0243c8e7a5f9f8c06254490f16#diff-a4f102bc7dbce569c0cbe3e1908e5c6f

Reply:
Can you explain what does not work correctly? What happens? Have you mentioned this issue upstream, considering building wheels is the way forward.

Reply:
@FRidh In `mailnag-config` plugins are not seen, then when closing `mailnag-config` it should start `mailnag`, but it can't with error `No such file or directory: 'IOError: [Errno 2] No such file or directory: '/usr/share/applications/mailnag.desktop'`. Then trying to send email myself - there should be libnotify notification but there is none. These things are not happening when these `buildPhase and installPhase` are here. This is the patching `setup.py` does https://github.com/pulb/mailnag/blob/master/setup.py#L22

Reply:
That's because of https://github.com/NixOS/nixpkgs/issues/23438.

Reply:
Which is an upstream issue https://github.com/pypa/setuptools/issues/130
(Note the name iElectric, that's '@'domenkozar )

Reply:
@FRidh Ok. What would be the temporary fix? Because at this stage the package is broken.